client/src/components/mainPage/mainPage.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Header from '../header/header.component';
+import createFooter from '../footer/footer.component';
+import MoviesList from '../moviesList/moviesList.component';
+import { title, releaseDate, rating } from '../../consts';
+import movies from '../../mock-data';
+import createSortBar from '../sortBar/sortBar.component';
+import { filterByTitle, filterByGenre, sortByDate, sortByRating } from '../../utils/filterMethods';
+import styles from './mainPage.css';
+
+class MainPage extends React.Component {
```

Comment:
Consider turning this component into container with JSX child components instead of function calls in the render method.
And maybe moving Header container outside of MainPage. App.js could be a good place to have a structured layout visible.